src/App.js

```diff
@@ -1,29 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div className="App">
+    <header className="App-header">
+      {calculator()}
+
+    </header>
+  </div>
+);
```

Comment:
`README.md` file:

- [ ] To make your documentation be more descriptive and professional, I think it would be better if you improve your `Getting started` section with correct anf all information needed by users(e.g.: non-technical users,...) such as of how to `clone` your repo, `access` the cloned directory, `run build` for production,... To help you, I suggest you refer this **[example file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**